src/Angor/Client/Pages/Invest.razor

```diff
@@ -613,6 +629,9 @@
             investorProject.SignaturesInfo!.SignatureRequestEventId = investmentSigsRequest.eventId;
 
             storage.AddInvestmentProject(investorProject);
+            
+            storage.SetInvestmentAmount(investorProject.ProjectInfo.ProjectIdentifier, Investment.InvestmentAmount);
```

Comment:
We have the type `InvestorProject` and it has the property `InvestorProject.AmountInvested`
Why can't we use this here? why is there a difference between the model and the storage type?

I see that the property is set when we actually publish to the blockchain in this method `investorProject.CompleteProjectInvestment()`  can we not use that property instead of saving to storage a temporary page param